src/patternfly/components/Label/label-editable-content.hbs

```diff
@@ -1,9 +1,10 @@
 <{{#if label-editable-content--type}}{{label-editable-content--type}}{{else if label--IsEditableActive}}input{{else}}button{{/if}} class="pf-c-label__content{{#if label-editable-content--modifier}} {{label-editable-content--modifier}}{{/if}}"
   id="{{label--id}}-editable-content"
-  {{#if label--IsEditableActive}} 
-    type="text" 
-    value="{{~> @partial-block}}"{{else}} 
-    currvalue="{{~> @partial-block}}"
+  {{#if label--IsEditableActive}}
+    type="text"
+    value="{{label-editable-content--value}}"
+  {{else}}
+    currvalue="{{label-editable-content--value}}"
```

Comment:
out of scope but I don't understand what this does? It's in the docs, but it isn't a valid html attr and isn't used in react. I don't think you need it - you'd just update the button text with the input value.

https://github.com/patternfly/patternfly/blob/bef73445bf939b7a022ba297a7697a3c555821ef/src/patternfly/components/Label/examples/Label.md?plain=1#L89